CMakeLists.txt

```diff
@@ -141,7 +144,7 @@ function(preprocess_def_file inputFilename outputFilename)
 
   add_custom_command(
     OUTPUT ${outputFilename}
-    COMMAND ${CMAKE_CXX_COMPILER} /P /EP /TC ${PREPROCESS_DEFINITIONS}  /Fi${outputFilename}  ${inputFilename}
+    COMMAND cl.exe /P /EP /TC ${PREPROCESS_DEFINITIONS}  /Fi${outputFilename}  ${inputFilename}
```

Comment:
Should `cl.exe' be really hardcoded? Same goes for other binaries that got converted to a hardcoded name with .exe.